run_golden_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This script is intended to be invoked in a github action. It will run all the
+# the golden tests that it can find.
+
+set -eEuo pipefail
+
+exit_status=0
+for template_dir in $(find . -name 'testdata' | grep -v '/.git/' | xargs dirname) ; do
+    echo "Running golden tests for $template_dir"
+    go run cmd/abc/abc.go templates golden-test verify $template_dir
```

Comment:
I wonder if this is an indication that we should have a built-in flag like `-recurse` and have the `golden-test` command handle the nesting?

Reply:
I agree. We have #311 for that, coming soon.